deployment/keystone/changeset/operations/contracts/add_capabilites_op.go

```diff
@@ -49,3 +54,80 @@ var AddCapabilitiesOp = operations.NewOperation[AddCapabilitiesOpInput, AddCapab
 		return AddCapabilitiesOpOutput{BatchOperation: batchOp}, nil
 	},
 )
+
+type AppendCapabilitiesOpDeps struct {
+	Env               *cldf.Environment
+	P2pToCapabilities map[p2pkey.PeerID][]capabilities_registry.CapabilitiesRegistryCapability
```

Comment:
Are we sure this cannot be an input instead? We don't want it public? Or cannot be serialized properly?